Sources/CoreMetrics/Metrics.swift

```diff
@@ -269,6 +274,84 @@ public final class Gauge: Recorder {
     }
 }
 
+// MARK: - Gauge2
+
+/// A gauge is a metric that represents a single numerical value that can arbitrarily go up and down.
+/// Gauges are typically used for measured values like temperatures or current memory usage, but also "counts" that can go up and down, like the number of active threads.
+// FIXME: better name please!
+public final class Gauge2 {
```

Comment:
Any good ideas given https://github.com/apple/swift-metrics/pull/123#discussion_r1153886760? Some names I came up with:
* GaugeDial
* GaugeMeter
* Gage

Reply:
I think I prefer `FlexibleGauge` to be honest.

Reply:
Agree with @Lukasa

Reply:
`FlexibleGauge` is the best one IMO.

Reply:
Gauge2 is hilarious and reminds me of the good ol' Guava days with `Collections2` 😉 

Seems people like "Flexible" so we can go with that 👍 
I had thought `PN` which is "Positive Negative" which comes from CRDT terminology, but it's too obscure probably 😉 

`FlexibleGauge` LGTM 👍

Reply:
`FlexibleGauge` makes me think of a Gauge made of rubber :D to be clear, I dont love any of the other names and Gauge2 is a placeholder not a actual proposal 

@ktoso you also brought `PNCounter` so maybe even `PNGauge`? I can live with that 

https://docs.hazelcast.org/docs/4.0/javadoc/com/hazelcast/crdt/pncounter/PNCounter.html

Reply:
FWIW, `PN` as a prefix is a bit opaque to me, so I'd be unlikely to detect this as "the right choice". But I don't hate it.

Reply:
Personally, I still prefer the `FlexibleGauge` over `PNGauge` naming. IMO it conveys the semantics more and the `PN` prefix reminds me of all the Objective-C types that got their module as a prefix.

Reply:
Just throwing in another name then `IncrementingDecrementingGauge` that really says everything that it does over `Gauge` but I still prefer `FlexibleGauge`

Reply:
As a nice parallel with `swift-metrics`, maybe `meter` instead of `gauge`?

A meter as a noun is "One who measures", like a parking meter, a metering pump, or a meter meter.

Reply:
I think 'Meter' could work. @ktoso @FranzBusch @Lukasa?

Reply:
Meter can work 👍

Reply:
I have new idea: `Gauger`: this rhymes with `Counter`, `Timer`, `Recorder, its close to the real name we want (`Gauge`) and it seems to be a real word: https://www.merriam-webster.com/dictionary/gauger

Reply:
A bit weird to be honest, Meter would be fine IMHO, if strong feelings for Gauger I guess we could -- it just sounds a bit weird/unknown

Reply:
I quite like `Meter`

Reply:
The challenge with picking Meter is that it would be odd to have Meter as the user facing type and GaugeHandler etc as the underlying machinery. As such, a term that is close to Gauge (which is the name we really want given it is the industry standard) would work best